pom.xml

```diff
@@ -257,7 +257,7 @@
     <version.org.hamcrest>1.3</version.org.hamcrest>
     <version.junit>4.12</version.junit>
     <!-- keep in sync with version.com.datastax.cassandra -->
```

Comment:
As per this comment, shouldn't they be in sync? Or should the comment be removed?